src/Repository/AnnouncementRepository.php

```diff
@@ -18,64 +18,77 @@ class AnnouncementRepository extends ServiceEntityRepository
   public function __construct(ManagerRegistry $registry)
   {
     parent::__construct($registry, Announcement::class);
+
   }
 
   /**
    * Custom method to pull all announcements created for the current date
    * 
    * @author Daniel Boling
    */
-  public function find_today()
+  private function query_today($date_input = 'now')
```

Comment:
Would it work to add an argument to this function definition like `$approved=true` and then use an `if` statement in the function to selectively add the `->andWhere('a.approval = 1')` condition?  You could do the same thing for the `find_today()` function which would remove the need for a separate function named `find_today_approved()`.  Does that make sense?

Reply:
Can I put a condition directly into the query line there, or would I need to make "two separate queries" in one function, just selected by a condition?